packages/core/package.json

```diff
@@ -1,10 +1,15 @@
 {
-  "name": "@aerogearservices/core",
+  "name": "aerogear-core",
```

Comment:
Is that ok for you @paolobueno ? I thought we want to use scoped packages?

Reply:
Will be best to leave scoped packages - this helps a lot with versioning and management.

Reply:
+1 for scoped, we haven't setup the scope in the main npm registry yet, but it is also a good way to distinguish these packages from the existing UPS sdks.

Reply:
There wasn't any particular reason, I will change it back.